### PR TITLE
locale_gen: fix UbuntuMode

### PR DESCRIPTION
**Pull request by Bartosz-lab**
##### SUMMARY

When I tried to use locales_gen module in my playbook to manage locales in Ubuntu 22.04 it did nothing.
Now it works properly.

#5281

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

locale_gen module

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by Bartosz-lab** [comment]
> Thanks for your contribution! Could you please add a [changelog fragment](https://docs.ansible.com/ansible/devel/community/collection_development_process.html#creating-a-changelog-fragment)? Thanks.

OK. I added it.
`changelogs/fragments/5281-locale_gen.yaml`

File: changelogs/fragments/5281-locale_gen.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - "locale_gen - fix ubuntumode (https://github.com/ansible-collections/community.general/issues/5281, https://github.com/ansible-collections/community.general/pull/5281)."
```
Comment (felixfontein) [inline changelogs/fragments/5281-locale_gen.yaml:2]:
```suggestion
  - "locale_gen - fix support for Ubuntu (https://github.com/ansible-collections/community.general/issues/5281, https://github.com/ansible-collections/community.general/pull/5281)."
```

Comment (Bartosz-lab) [reply changelogs/fragments/5281-locale_gen.yaml]: OK. I've done it.
`4b581b5b349de7df1b4d3231a99d33f9caa60a47`
:)

File: changelogs/fragments/5282-locale_gen.yaml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - "locale_gen - fix support for Ubuntu (https://github.com/ansible-collections/community.general/issues/5281, https://github.com/ansible-collections/community.general/pull/5282)."
```
Comment (russoz) [inline changelogs/fragments/5282-locale_gen.yaml:2]: When there is an issue the PR is solving, we usually refer only to the issue, not the PR:
```suggestion
  - "locale_gen - fix support for Ubuntu (https://github.com/ansible-collections/community.general/issues/5281)."
```
This is not a hard requirement. Feel free to ignore if you prefer it like that.

**Review by russoz — APPROVED** [review]
Small optional adjustment, other than that LGTM.

File: changelogs/fragments/5282-locale_gen.yaml (hunk 2 above)
Comment (Bartosz-lab) [reply changelogs/fragments/5282-locale_gen.yaml]: Done

Comment (felixfontein) [reply changelogs/fragments/5282-locale_gen.yaml]: I usually prefer to mention both issue and PR, but that's individual preference.

**Comment by felixfontein** [comment]
@Bartosz-lab thanks for fixing this!
@russoz thanks for reviewing!

**Pull request merged**